addons/account/populate/account_reconcile_model.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models
+from odoo.tools import populate
+
+import logging
+
+_logger = logging.getLogger(__name__)
+
+class AccountReconcileModel(models.Model):
+    _inherit = "account.reconcile.model"
+
+    _populate_sizes = {
+        'small': 5,
+        'medium': 100,
+        'large': 1000,
+    }
+
+    _populate_dependencies = ['res.company']
+
+    def _populate_factories(self):
+        def get_name(counter, **kwargs):
+            return 'model_%s' % counter
+
+        company_ids = self.env['res.company'].search([('chart_template_id', '!=', False), ('id', 'in', self.env.registry.populated_models['res.company'])])
+        return [
+            ('company_id', populate.cartesian(company_ids.ids)),
+            ('rule_type', populate.cartesian(['writeoff_button', 'writeoff_suggestion'])),
+            # ('auto_reconcile', populate.cartesian([True, False], [0.1, 0.9])),
+            ('name', populate.compute(get_name)),
+        ]
```

Comment:
not sure of the usefulness of that part, but hey.. why not..

Reply:
To have something to work with when opening the reconciliation widget, otherwise it would be a bit empty.